server/errors.go

```diff
@@ -22,3 +24,14 @@ func (me MetaError) Error() string {
 func (me MetaError) Unwrap() error {
 	return me.Err
 }
+
+func is400(err error) bool {
+	// proxy requests are super simple (clients basically only pass bytes), so the only 400 possible
+	// is passing a blog that's too big.
+	//
+	// Any 400s returned by the disperser are due to formatting bugs in proxy code, for eg. badly
```

Comment:
another common 400 error is rate limit. Or do you think it should be 201?

Reply:
see related comment here, https://github.com/Layr-Labs/eigenda/pull/828/files#r1821614539

Reply:
400 here means only 400, not 4xx. This made me realize I was forgetting about 429s though, so added them in [8dbc072](https://github.com/Layr-Labs/eigenda-proxy/pull/193/commits/8dbc07208968b96bacdbf5276323b76c27468ad2)

Reply:
yeah, that is what I mean